src/appliance-onboarding-script/appliance_setup/avs/avsarconboarder/executor/process/process_executor.py

```diff
@@ -21,6 +23,11 @@ def execute(self, *args):
             logging.info(self._process_cmd+" >> "+log_path)
         else:
             process_cmd = self._process_cmd
-        output = subprocess.check_output(process_cmd, timeout=100000, shell=True)
-        result = output.decode('UTF-8', errors='strict')
+        try:
+            output = subprocess.check_output(process_cmd, timeout=100000, shell=True)
+        except subprocess.CalledProcessError as e:
+            logging.error(bytes_to_string(e.output))
+            return None
+        result = bytes_to_string(output)
```

Comment:
Check if result can be computed first and returned as part of exception handling also.